src/vivarium/framework/engine.py

```diff
@@ -343,13 +354,13 @@ def add_components(self, component_list: list[Component]) -> None:
         self._component_manager.add_components(component_list)
 
     def get_population(self, untracked: bool = True) -> pd.DataFrame:
-        return self._population.get_population(untracked)
+        return pd.DataFrame(self._population.get_population(untracked))
```

Comment:
Why is this necessary? `get_population()` is already type-hinted to return a DataFrame.